### ISSUE TITLE
Idea: planning mode as shortcut

### ISSUE THREAD

**Issue by nilsalm** [nilsalm/satsuma#79]
I think the planning mode / shopping mode is a really nice feature, but in some places it just adds an extra step. In the /categories page I have already removed it because there's no point to that page in an non-editable state. 

I got this idea that the middle button where we toggle the planning mode could instead **take you straight to the last opened list** and **enable planning mode**.

Pressing that button again should then disable planning mode, just as before. And if the user wants to go to a different list they can press the LISTS-button or we could provide a back or up button. 

My thinking is that it could make the everyday use even more streamlined, where the user probably spends most time in one list but might jump to /categories or a template or so, but then wants to return to the list again that they worked on. 

What do you think @brokelyn ?

**Comment by brokelyn** [comment]
yes I like that idea. I already had the feeling that it might not work on every page so that new usage of the button would be awesome!
